typings/objection/index.d.ts

```diff
@@ -196,6 +196,26 @@ declare namespace Objection {
     [K in DataPropertyNames<T>]: T[K];
   };
 
+  /**
+   * Returns keys that have function as their type.
+   */
+  type KeysWithFunctionType<T> = {
+    [K in keyof T]: T[K] extends Function ? K : never;
+  }[keyof T];
```

Comment:
This is the reverse of the current `NonFunctionPropertyNames`, I think the name should reflect that:

`FunctionPropertyNames`